RestaurentManagement/src/main/java/com/restapi/services/MenuService.java

```diff
@@ -45,8 +50,14 @@ public Menu createMenu(Menu menu) {
         return menuRepository.save(menu);
     }
 
-    public List<Menu> getAllMenus() {
-        return menuRepository.findAll();
+    public List<Menu> getAllMenusPaged(int page, int size, String sortBy) {
+        page = RMUtils.setPage(page,size, (int) menuRepository.count());
+        Pageable pageable = PageRequest.of(page-1, size, Sort.by(Sort.Order.desc(sortBy)));
```

Comment:
Can you calculate the true page in setPage() instead of calling setPage() then -1? I think that way to more clean